src/main/java/fr/xephi/authme/service/PluginHookService.java

```diff
@@ -60,6 +62,19 @@ public File getEssentialsDataFolder() {
         return null;
     }
 
+    /**
+     * If CMI is hooked into, return CMI' data folder.
+     *
+     * @return The CMI data folder, or null if unavailable
+     */
+    public File getCMIDataFolder() {
```

Comment:
For here and all the other methods—could we rename `CMI` in variables and method names to `Cmi`? 

This would be in line with the Google styleguide https://google.github.io/styleguide/javaguide.html#s5.3-camel-case